ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml

```diff
@@ -0,0 +1,216 @@
+# binary_build_commands: NVIDIA=1 RUN_TESTS=0 BUILDAH_FORMAT=docker OV_USE_BINARY=0
+#   BASE_OS=redhat RH_ENTITLEMENT=1 RH_ENTITLEMENT_DIR=$(pwd)/entitlement OV_SOURCE_BRANCH='releases/2022/3'
+#   OV_CONTRIB_BRANCH='releases/2022/3' make docker_build
+
+# The above invokes docker build commands, is that possible? If not, the below replicates most of what the make command does
+
+build_root:
+  from_repository: true # Need some clarification for this part
+
+# Entitlements, is the cluster able to build with entitlements or no? Need to figure that out since there's 4 or 5 packages not in UBI repos that we need from RHEL/CodeReady Builder
+
+images: # Does this allow dependencies?
+  - dockerfile_literal: |- # Guessing this is how one would do the equivalent of a shell cp in the source directory?
+      FROM ubi8/ubi
+      COPY src/queue.hpp /
+    inputs:
+      bin:
+        paths:
+          - destination_dir: /src/custom_nodes/
```

Comment:
This is how you will copy the binary from the `bin` image to the current one.

Reply:
https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-container-images